lib/api.go

```diff
@@ -12,12 +12,16 @@ import (
 
 func AddAPIHandlers(s *sebakstorage.LevelDBBackend) func(ctx context.Context, t *sebaknetwork.HTTP2Network) {
 	fn := func(ctx context.Context, t *sebaknetwork.HTTP2Network) {
-		t.AddAPIHandler("/account/{address}", GetAccountHandler(s)).Methods("GET")
+		t.AddAPIHandler(GetAccountHandlerPattern, GetAccountHandler(s)).Methods("GET")
+		t.AddAPIHandler(GetAccountTransactionsHandlerPattern, GetAccountTransactionsHandler(s)).Methods("GET")
+		t.AddAPIHandler(GetAccountOperationsHandlerPattern, GetAccountOperationsHandler(s)).Methods("GET")
+		t.AddAPIHandler(GetTransactionByHashHandlerPattern, GetTransactionByHashHandler(s)).Methods("GET")
```

Comment:
I want to put the pattern and its handler together.
That is the reason why I use constant here.
Btw, we are planing to refactor APIs and handlers, at that time the point you are considering may be solved.